bundle/Matcher/MatcherFactory.php

```diff
@@ -54,4 +67,27 @@ public function match(View $view): ?array
 
         return parent::match($view);
     }
+
+    /**
+     * @param string $matcherIdentifier
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     *
+     * @return MatcherInterface|ViewMatcherInterface
+     */
+    protected function getMatcher($matcherIdentifier)
+    {
+        if ($this->viewMatcherRegistry !== null && \strpos($matcherIdentifier, '@') === 0) {
+            return $this->viewMatcherRegistry->getMatcher(\substr($matcherIdentifier, 1));
+        }
+
+        if ($this->container->has($matcherIdentifier)) {
```

Comment:
Do we need this still? This is for eZ Platform 2.5 support?

Reply:
Yes, in case someone references a matcher by service name that does not begin with `@`.